docs-js_versioned_docs/version-v1/ai-core/document-grounding.mdx

```diff
@@ -88,6 +129,72 @@ const response: DocumentsListResponse = await VectorApi.createDocuments(
 ).execute();
 ```
 
+### Retrieval Search
+
+See the following code sample for searching document grounding data with a query.
+Please refer to [Retrieval API documentation](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/retrieval-api) for more detail.
+
+```ts
+const response: RetrievalSearchResults = RetrievalApi.search(
+  {
+    query: 'What is SAP Cloud SDK for AI?',
+    filters: [
+      {
+        id: 'my-filter',
+        searchConfiguration: {
+          maxChunkCount: 10
+        },
+        dataRepositories: ['*'],
+        dataRepositoryType: 'vector'
+      }
+    ]
+  },
+  {
+    'AI-Resource-Group': 'default'
+  }
+).execute();
+```
+
+### Orchestration Grounding
+
+Below, you will find the sample code on how you can use the grounding service via orchestration.
+See also [grounding on orchestration](https://sap.github.io/ai-sdk/docs/js/orchestration/chat-completion#grounding).
+
+```ts
+const orchestrationClient = new OrchestrationClient(
+  {
+    // define the language model to be used
+    llm: {
+      model_name: 'gpt-4o'
+    },
+    grounding: buildDocumentGroundingConfig({
+      input_params: ['groundingRequest'],
+      output_param: 'groundingOutput',
+      filters: [
+        {
+          data_repository_type: 'vector',
+          data_repositories: ['*']
+        }
+      ]
+    })
+  },
+  { resourceGroup: 'default' }
+);
+
+const chatResponse = await orchestrationClient.chatCompletion({
+  messages: [
+    {
+      role: 'user',
+      content:
+        'UserQuestion: {{?groundingRequest}} Context: {{?groundingOutput}}'
+    }
+  ],
+  inputParams: {
+    groundingRequest: 'What is SAP Cloud SDK for AI?'
+  }
+});
+```
```

Comment:
I don't think this part is needed. Just link them to the orchestration client doc.